Doc/whatsnew/3.12.rst

```diff
@@ -1274,29 +1274,41 @@ although there is currently no date scheduled for their removal.
 Removed
 =======
 
-* ``asynchat`` and ``asyncore``: These two modules have been removed
+asynchat and asyncore
+---------------------
+
+* These two modules have been removed
   according to the schedule in :pep:`594`,
   having been deprecated in Python 3.6.
   Use :mod:`asyncio` instead.
   (Contributed by Nikita Sobolev in :gh:`96580`.)
 
-* :mod:`configparser`: Several names deprecated in the :mod:`configparser` way back in 3.2 have
-  been removed per :gh:`89336`:
+configparser
+------------
+
+Several names deprecated in the :mod:`configparser` way back in 3.2 have
+been removed per :gh:`89336`:
```

Comment:
Should we use bullets for these new sections? E.g. async* above does.

Reply:
I kept the original bullets for most of these sections, that only had a single level of bullets.

A few (at least this, importlib and unittest) were of the form:

---

* configparser: Several names deprecated ... have been removed:

  * first one
  * second
  * third
---

And I changed them to:

---

## configparser

Several names deprecated ... have been removed:

* first one
* second
* third

---

But on second thoughts, it's probably better to keep the original two-level bullets for consistency, so things are all nicely aligned as you scroll down?

Reply:
I'd agree--no strong opinions on keeping / removing, it was just the consistency point.

A

Reply:
Thanks, reindented.